fixtures/range_with_no_upper_bound_actual.rb

```diff
@@ -0,0 +1 @@
+(1..)
```

Comment:
I guess we could place this under `fixtures/2.6` to only have it run under 2.6?

Reply:
Hi, Teo! I think that makes sense, but it's also the reason we'll need to change the test script: `fixtures/2.6` is currently gitignored, because the current test script copies `fixtures/2.5` to `fixtures/2.6`.